### PR TITLE
Correction of layer names

### PR DESCRIPTION
**Pull request by mikehaller**
Fixing the layer names so that the dependency check finds the correct names

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Build folder is generated by kas tool
+build
```
Comment (stlachev) [inline .gitignore:2]: Please, add additional folders

**Review by stlachev — CHANGES_REQUESTED** [review]
Please, fix .gitignore

File: .gitignore (hunk 1 above)
Comment (mikehaller) [reply .gitignore]: Fixed and added the modules which are handled by kas so that they're ignored by git.

**Review by mikehaller — COMMENTED** [review]
Fixed minor issue

**Review by stlachev — APPROVED** [review]
sgtm

**Pull request merged**